source/jest-puppeteer.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  launch: {
+    headless: process.env.CI === 'true'
+  },
+  server: {
+    // TODO: insure http-server is in path
+    command: 'node server.js',
+    port: 8000,
+    launchTimeout: 4000
```

Comment:
I'm not sure if we really need this server config.

Reply:
The default is 5000, so IMO we can remove this.

Reply:
Will remove.